gateway/test/dapi_crud_test.go

```diff
@@ -529,28 +531,6 @@ func (s *GatewayOpsTestSuite) TestDapiPost() {
 		})
 	})
 
-	// ING-1110
```

Comment:
Why are we deleting these tests instead of uncommenting?

Reply:
The tests that are changed above actually cover this specific case already.